python/README.md

```diff
@@ -0,0 +1,95 @@
+# Python
+
+Python hits a sweet spot between short shell scripts and large applications.
+We use Python 3.6+ for largish scripts and smallish libraries.
+
+## Links
+
+- [Python 3 documentation](https://docs.python.org/3/)
+    - `!py` with [DuckDuckGo](https://duckduckgo.com)
+
+## Setup
+
+Use [`venv`](https://docs.python.org/3/library/venv.html?highlight=venv#module-venv)
+to isolate your Python version and libraries from other projects.
+We standardize on a `_venv` directory:
```

Comment:
I'm not sure I agree with having just one global `_venv` directory. I have many different virtual environments that I use for different repositories.

Reply:
Sorry, I meant this as a per-project `_venv` subdirectory, not a global/shared `~/_venv`.

Reply:
Ok, that makes sense.

Reply:
I clarified this.